packages/react/src/components/Form/Form.tsx

```diff
@@ -100,6 +100,6 @@ class Form extends UIComponent<WithAsProp<FormProps>, any> {
 }
 
 /**
- * A Form displays a set of related user input fields in a structured way.
+ * A Form collects, validates, and submits the user input, in a structured way.
```

Comment:
We don't do any validation at present.... A Form is used to collect, optionally validate and submit the user input in a structured way?